src/any_llm/providers/otari/otari.py

```
@@ -57,6 +61,19 @@ class ListBatchesOptions(TypedDict, total=False):
 LEGACY_GATEWAY_API_BASE_ENV = "GATEWAY_API_BASE"
 
 
+def _as_plain_dict(response: Any) -> Any:
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `response`

(ANN401)